components/shared/gallery/BreadcrumbsFilter.vue

```diff
@@ -34,7 +34,8 @@ const breads = computed(() => {
 
   const activeFilters = Object.entries(query).filter(
     ([key, value]) =>
-      (key === 'search' && value !== undefined) || value === 'true'
+      (key === 'search' && value !== undefined && value !== '') ||
```

Comment:
How about this instead?
```suggestion
(key === 'search' && Boolean(value)) ||
```